Variable is not defined in test_dhcp_relay.py module

<!--
If you are reporting a new issue, make sure that we do not have any duplicates
already open. You can ensure this by searching the issue list for this
repository. If there is a duplicate, please close your issue and add a comment
to the existing issue instead.

If you suspect your issue is a bug, please edit your issue description to
include the BUG REPORT INFORMATION shown below. If you fail to provide this
information within 7 days, we cannot debug your issue and will close it. We
will, however, reopen it if you later provide the information.

For more information about reporting issues, see
https://github.com/Azure/SONiC/wiki#report-issues

---------------------------------------------------
GENERAL SUPPORT INFORMATION
---------------------------------------------------

The GitHub issue tracker is for bug reports and feature requests.
General support can be found at the following locations:

- SONiC Support Forums - https://groups.google.com/forum/#!forum/sonicproject

---------------------------------------------------
BUG REPORT INFORMATION
---------------------------------------------------
Use the commands below to provide key information from your environment:
You do NOT have to include this information if this is a FEATURE REQUEST
-->

**Description**
If there will be no route to DHCP server during run any of "sonic-mgmt/tests/dhcp_relay/test_dhcp_relay.py" test cases , failure reasone message will be displayed, but will fail because of variable "result" is not defined.

    @pytest.fixture(scope="module") 
    def validate_dut_routes_exist(duthost, dut_dhcp_relay_data): 
        """Fixture to valid a route to each DHCP server exist 
        """ 
        dhcp_servers = set() 
        for dhcp_relay in dut_dhcp_relay_data: 
            dhcp_servers |= set(dhcp_relay['downlink_vlan_iface']['dhcp_server_addrs']) 

        for dhcp_server in dhcp_servers: 
            rtInfo = duthost.get_ip_route_info(ipaddress.ip_address(dhcp_server)) 
            assert len(rtInfo["nexthops"]) > 0, \ 
>               "Failed to find route to DHCP server '{0}' with error '{1}".format(dhcp_server, result["stderr"]) 
E           NameError: global name 'result' is not defined 

<!--
Briefly describe the problem you are having in a few paragraphs.
-->

**Steps to reproduce the issue:**
1. Run "sonic-mgmt/tests/dhcp_relay/test_dhcp_relay.py" test cases

**Describe the results you received:**


**Describe the results you expected:**


**Additional information you deem important:**
<!--
software/ASIC/Hardware Flatform version and info
-->
    **Output of `show version`:**

    ```
    (paste your output here)
    ```

    **Attach debug file `sudo generate_dump`:**

    ```
    (paste your output here)
    ```

--- a/tests/dhcp_relay/test_dhcp_relay.py
+++ b/tests/dhcp_relay/test_dhcp_relay.py
@@ -85,8 +85,7 @@ def validate_dut_routes_exist(duthost, dut_dhcp_relay_data):
 
     for dhcp_server in dhcp_servers:
         rtInfo = duthost.get_ip_route_info(ipaddress.ip_address(dhcp_server))
-        assert len(rtInfo["nexthops"]) > 0, \
-            "Failed to find route to DHCP server '{0}' with error '{1}".format(dhcp_server, result["stderr"])
+        assert len(rtInfo["nexthops"]) > 0, "Failed to find route to DHCP server '{0}'".format(dhcp_server)
 
 
 def test_dhcp_relay_default(duthost, ptfhost, dut_dhcp_relay_data, validate_dut_routes_exist):

